.dumi/theme/builtins/ComponentOverview/index.tsx

```
@@ -1,7 +1,19 @@
 import { SearchOutlined } from '@ant-design/icons';
-import { Affix, Card, Col, Divider, Flex, Input, Row, Tag, Typography } from 'antd';
+import {
+  Affix,
+  BorderBeam,
+  Card,
+  Col,
+  Divider,
+  Flex,
+  Input,
+  Row,
+  Skeleton,
```

ESLint

[error] 'Skeleton' is defined but never used.

(unused-imports/no-unused-imports)